base_sdm845_defconfig

```diff
@@ -550,3 +550,9 @@ CONFIG_SND_SOC_MSM_SDW=y
 CONFIG_SND_SOC_MSM_HDMI_CODEC_RX=y
 CONFIG_SND_SOC_SDM845=y
 CONFIG_SND_SOC_WCD_CPE_SOMC_EXT=y
+CONFIG_CPU_QUIET=y
+CONFIG_CPU_QUIET_MSM=y
+CONFIG_CPU_QUIET_DEFAULT_GOV_RQBALANCE=y
+# Not maintained, requires forward porting to newer kernels
+CONFIG_CPU_QUIET_GOVERNOR_RUNNABLE=n
```

Comment:
The kernel standard is to use `# CONFIG_CPU_QUIET_GOVERNOR_RUNNABLE is not set`.

Reply:
That is the standard indeed, and I have verified that is how it will be propagated to the derived defconfigs.

For this one, which is used as base for others, I thought it would make more sense to provide additional details so that other people would be aware of the intentions ;)

Reply:
You can leave the comment if you feel like it (though I have not validated this, so this is not a formal "ack"), but the `=n` syntax should be replaced with `# is not set`.

Reply:
I see what you mean now, valid point, thank you